src/Commands/SendTelemetryEvent.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Enjin\Platform\Commands;
+
+use Enjin\Platform\Enums\TelemetrySource;
+use Enjin\Platform\Services\PhoneHomeService;
+use Illuminate\Console\Command;
+
+class SendTelemetryEvent extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'platform:send-telemetry-event';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Send telemetry event to Enjin';
+
```

Comment:
```suggestion
    protected $description = 'Send telemetry event to Enjin';

    protected $hidden = true;

```

To prevent bloating the help command with meaningless commands, we can probably hide this command from being rendered when calling `artisan help`. The fact that this remains as a standalone command remains beneficial for the start script.